packages/client/src/views/Settings/items/PhoneNumber.tsx

```diff
@@ -0,0 +1,94 @@
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import * as React from 'react'
+import { useIntl, FormattedMessage } from 'react-intl'
+import {
+  LabelContainer,
+  ValueContainer,
+  DynamicHeightLinkButton
+} from '@client/views/Settings/items/utils'
+import { useSelector, useDispatch } from 'react-redux'
+import { IStoreState } from '@client/store'
+import {
+  constantsMessages,
+  buttonMessages,
+  userMessages
+} from '@client/i18n/messages'
+import {
+  ListViewItemSimplified,
+  FloatingNotification,
+  NOTIFICATION_TYPE
+} from '@opencrvs/components/lib/interface'
+import { withOnlineStatus } from '@client/views/OfficeHome/LoadingIndicator'
+import { goToPhoneSettings } from '@client/navigation'
+import { useHistory } from 'react-router'
+import { SETTINGS } from '@client/navigation/routes'
+
+export function PhoneNumberCommponent({ isOnline }: { isOnline: boolean }) {
+  const intl = useIntl()
+  const mobile = useSelector<IStoreState, string>(
+    (state) => state.profile.userDetails?.mobile || ''
+  )
+  const dispatch = useDispatch()
+  const onClickChange = React.useCallback(
+    () => dispatch(goToPhoneSettings()),
+    [dispatch]
+  )
+  const history = useHistory()
+  const [showSuccessNotification, setShowSuccessNotification] =
+    React.useState<boolean>(false)
+  const toggleSuccessNotification = React.useCallback(() => {
+    setShowSuccessNotification((prevValue) => !prevValue)
+  }, [])
+
+  React.useEffect(() => {
+    if (history && history.location.state) {
+      let phonedNumberUpdated = false
+      const historyState = history.location.state as Record<string, boolean>
+      phonedNumberUpdated = historyState.phonedNumberUpdated
+      if (phonedNumberUpdated) {
+        toggleSuccessNotification()
+        history.replace({
+          pathname: SETTINGS,
+          state: { phonedNumberUpdated: false }
+        })
+      }
+    }
+  }, [history, toggleSuccessNotification])
+
+  return (
+    <>
+      <ListViewItemSimplified
+        label={
+          <LabelContainer>
+            {intl.formatMessage(constantsMessages.labelPhone)}
+          </LabelContainer>
+        }
+        value={<ValueContainer>{mobile}</ValueContainer>}
+        actions={
+          <DynamicHeightLinkButton onClick={onClickChange} disabled={!isOnline}>
+            {intl.formatMessage(buttonMessages.change)}
+          </DynamicHeightLinkButton>
+        }
+      />
+      <FloatingNotification
+        type={NOTIFICATION_TYPE.SUCCESS}
+        show={showSuccessNotification}
+        callback={toggleSuccessNotification}
+      >
+        <FormattedMessage {...userMessages.phoneNumberUpdated} />
+      </FloatingNotification>
+    </>
+  )
+}
+
+export const PhoneNumber = withOnlineStatus(PhoneNumberCommponent)
```

Comment:
Could we create a `useOnlineStatus` hook now that we're gradually migrating to FCs?

Reply:
Thanks @Zangetsu101 , done in https://github.com/opencrvs/opencrvs-core/pull/3647/commits/fa6f52009d5b94033f2ca6dae0ff8e4f4e585ea2